pages/linux/repo-remove.md

```diff
@@ -0,0 +1,8 @@
+# repo-remove
+
+> Package database maintenance utility which removes packages from a local repository.
```

Comment:
```suggestion
> Package database maintenance utility which removes packages from the local repository.
```

Reply:
Since you can specify the repository file, I feel that "a" is more appropriate. Technically you could have multiple.